style.css

```diff
@@ -2,14 +2,197 @@ body {
   min-height: 100%;
   background-color: #0e0e0e;
   font-family: 'Roboto', sans-serif;
+  -webkit-user-select: none;
+  user-select: none;
 }
 
 .mainContainer {
   display: flex;
   flex-direction: column;
   margin: 0 auto;
   justify-items: center;
-  width: 60%;
+  width: 100%;
+  position: relative;
   background-color: #ffffff;
-  height: 100vh;
+  font-size: 16px;
+}
+
+.menu {
+  display: flex;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  margin-bottom: 1em;
```

Comment:
Are you both comfortable using em units? If the answer is no please change it to px.

Reply:
@bulic

Reply:
Yes